libs/langchain/langchain/vectorstores/redis/base.py

```diff
@@ -707,9 +707,14 @@ def add_texts(
 
         # Write data to redis
         pipeline = self.client.pipeline(transaction=False)
+        if not embeddings:
+            embeddings = []
+            for i, text in enumerate(texts):
+                embeddings.append(self._embeddings.embed_query(text))
+        self._create_index(len(embeddings[0]))
```

Comment:
The workflow is technically to use ``from_documents`` and then ``add_texts`` but this is the second time I've heard this feedback (https://github.com/langchain-ai/langchain/issues/9585#issuecomment-1716729646) so I'm starting to think people just use ``add_texts``.